potodo/json.py

```diff
@@ -3,6 +3,9 @@
 
 
 def json_dateconv(o: object) -> Optional[str]:
+    """
+    Convert an object to json.
```

Comment:
This is incorrect, this function is passed to the json module to convert a datetime object to a string